build.gradle

```diff
@@ -66,13 +66,13 @@ wrapper {
 ext {
     moduleName = "io.aiven.kafka.connect.http"
 
-    avroVersion = "1.8.1" // Version 1.8.1 brings Jackson 1.9.x/org.codehaus.jackson package for Avro and Confluent Platform 4.1.4.
-    confluentPlatformVersion = "4.1.4" // For compatibility tests use version 4.1.4.
+    avroVersion = "1.10.0" // Version 1.8.1 brings Jackson 1.9.x/org.codehaus.jackson package for Avro and Confluent Platform 4.1.4.
+    confluentPlatformVersion = "6.2.2" // For compatibility tests use version 6.2.2 to match Kafka 2.8.1.
     hamcrestVersion = "2.2"
     jacksonVersion = "2.13.2" // This Jackson is used in the tests.
     jupiterVersion = "5.8.2"
-    kafkaVersion = "2.0.1" // Oldest version supported, new versions are backwards compatible.
-    jettyVersion = "9.4.11.v20180605"
+    kafkaVersion = '2.8.1'
```

Comment:
Missed this somehow before but the PR is stating that you are updating the Kafka version to 2.6 and here it's 2.8.1. Should not it be 2.6.0?

Reply:
Back in March I was trying to figure out build failures and ended up moving everything to the latest Kafka 2.X version to solve it. This time around I figured out what was causing the dependency issues with Kafka 2.6.